java/code/src/com/redhat/rhn/frontend/action/LoginAction.java

```diff
@@ -64,22 +64,24 @@ public ActionForward execute(ActionMapping mapping,
         addErrors(request, errors);
         errors.clear();
 
+        String userAgent = request.getHeader("user-agent");
+
         // External-auth didn't return a user - try local-auth
         if (user == null) {
             user = loginUser(f.getString("username"), f.getString("password"),
                     request, response, errors);
             if (errors.isEmpty()) {
                 // No errors, log success
-                log.info("LOCAL AUTH SUCCESS: [" + user.getLogin() + "]");
+                log.info("LOCAL AUTH SUCCESS: [" + user.getLogin() + " - " + userAgent + "]");
```

Comment:
Don't you think it makes more sense to have the `userAgent` logged into `rhn_web_frontend.log`?

Reply:
I also thought about that, but I didn't want to pollute the logs since the user-agent shouldn't change per session.

But yeah, I can definitely change it to the frontend-logs :+1:

Reply:
I will do it

Reply:
@ncounter  done